_posts/2021-12-31-gs_00005_environment_variables.md

```diff
@@ -156,7 +156,7 @@ Hmm ... It is the same script as we used in the last post and there it worked.
 What went wrong? Can you guess? The answer is: we have overwritten the `PATH`
 variable so that it solely consists of your home directory. Since firefox is not
 located there, it cannot be found. By the way, also none of the Linux commands
-we've learned will not work anymore! Have a try:
```

Comment:
I think the previous was grammatically correct, wasn't it?

Reply:
none ... wil not? I think it is either:
"also the Linux commands we've learned will not work anymore" or
"also none of the ... will work anymore" so just one negative part

Reply:
Ohh, I missed the "none"!